### ISSUE TITLE
Incorrect Unirule mapping: UR000370960

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-annotation#5258]
**UniProt UniRule (ID and label):**

could you check this mapping? I might be missing something, does it have a protein substrate?
GO:0016540 | protein autoprocessing | IEA with UR000370960 | GO_REF:0000104

**Sequences with problematic annotation (ID + gene/protein name):**

**Description of issue**

**Comment by PedroRaposo** [comment]
This is a HAMAP rule. @pedruzzi could you please have a look?

**Comment by pedruzzi** [comment]
yes it has a protein substrate, itself. The enzyme is autocatalytically processed from a proenzyme to the mature form. Hence the GO terms for protein autoprocessing. This has nothing to do with its function as a phosphatidylserine decarboxylase, of course, and one may debate the use of this term to describe the protein function or the processes it is involved in. On the other hand, I don't see any other use case for this term as it is exactly what it describes.

**Comment by ValWood** [comment]
of course, I didn't know phosphatidylserine decarboxylase was autocatalytic. Thanks
v

**Comment by ValWood** [comment]
note to self,
annotate with substrate and remove filter
